layouts/partials/site-nav.html

```diff
@@ -1,5 +1,7 @@
 <nav class="relative max-w-7xl mx-auto flex items-center justify-between text-white px-4 sm:px-6 z-[9999] py-5" aria-label="Global">
-    <a href="/" class="text-3xl font-bold">ROTATIONAL LABS</a>
+    <a href="/">
+        <img src="img/logo-text.png" alt="Rotational Labs log" class="h-12 sm:h-14 w-auto">
```

Comment:
I cropped the text from the logo to see how it would look here. We can replace this with text and use the font used in the logo once it's purchased.